rc_building_model/htuse.py

```diff
@@ -122,3 +176,34 @@ def calculate_heat_loss_per_year(
     )
     heat_loss_kwh_for_heating_months.index = new_index
     return heat_loss_kwh_for_heating_months.sum(level=0).round()
+
+
+def calculate_useful_gains_per_year(
+    floor_area,
+    window_area,
+    mean_monthly_solar_gains=MEAN_MONTHLY_SOLAR_GAINS,
+    light_gain_value=8,  # Mean reference value taken from DEAP as 8W/m2 of gains
+):
+
+    solar_gain_kwh = (window_area / 6) * (
+        mean_monthly_solar_gains * UTILISATION_FACTOR
+    ).sum()  # Total Area divided by 6 to divide orientation
+    solar_gain_w = solar_gain_kwh * 1000 / 24
+    light_gains = floor_area * light_gain_value
+
+    return solar_gain_w + light_gains
+
+
+# Heating season defined as October-May as per DEAP
+
+
+def calculate_heat_use(
+    monthly_heat_loss,
+    monthly_useful_gains,
+):
+
+    monthly_heat_use_w = (
+        monthly_heat_loss - monthly_useful_gains
+    ) / 0.913  # Assume boiler efficiency of 91.3%
+
+    return (monthly_heat_use_w / 1000) * 24 * 243  # Calculating heat use in kWh
```

Comment:
```suggestion
    w_to_kw = (1 / 1000) 
    NAME_ME = 24 * 243
    return monthly_heat_use_w * w_to_kw  * NAME_ME
```

Better to capture the conversion factor with a variable or that better explains the purpose of each factor